### PR TITLE
27.see schedule olexiy

### PR DESCRIPTION
**Pull request by Karonuk**

### CODE REVIEW

File: src/app/components/main-schedule/main-schedule.component.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  GetScheduleByClass = (id: number) => {
+    let url = apiUrl + "/Schedule/GetScheduleByClass";
+    this.httpOptions.loadHeaders();
+    this.client.get(url + "/" + id, this.httpOptions.options).subscribe( (res: IScheduleDay[]) => {
+      this.days = res;
+    })
+  }
+
+  Reset = () => {
+    if(this.mode&&this.days!=null){
```
Comment (DmMov) [inline src/app/components/main-schedule/main-schedule.component.ts]: 1

File: src/app/components/main-schedule/main-schedule.component.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      {
+        this.mode = true;
+        this.classes = res;
+      });
+  }
+
+  GetScheduleByClass = (id: number) => {
+    let url = apiUrl + "/Schedule/GetScheduleByClass";
+    this.httpOptions.loadHeaders();
+    this.client.get(url + "/" + id, this.httpOptions.options).subscribe( (res: IScheduleDay[]) => {
```
Comment (DmMov) [inline src/app/components/main-schedule/main-schedule.component.ts]: 2

File: src/app/components/main-schedule/main-schedule.component.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+})
+export class MainScheduleComponent implements OnInit {
+  days: IScheduleDay[]
+  classes: IClassToSelect[]
+  classNumber = [1,2,3,4,5,6,7,8,9,10,11,12]
+  mode = false
+
+  constructor(
+    private client: HttpClient,
+    private httpOptions: HttpOptionsService,) { }
```
Comment (DmMov) [inline src/app/components/main-schedule/main-schedule.component.ts]: 3

File: src/app/components/main-schedule/main-schedule.component.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+@Component({
+  selector: 'yps-main-schedule',
+  templateUrl: './main-schedule.component.html',
+  styleUrls: ['./main-schedule.component.scss']
+})
+export class MainScheduleComponent implements OnInit {
+  days: IScheduleDay[]
+  classes: IClassToSelect[]
+  classNumber = [1,2,3,4,5,6,7,8,9,10,11,12]
+  mode = false
```
Comment (DmMov) [inline src/app/components/main-schedule/main-schedule.component.ts]: 4

File: src/app/components/main-schedule/main-schedule.component.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<h3 class="header">School Schedule</h3>
+<div class="selected" *ngIf="classes" (click)="Reset()">
+  <button class="btn">Back</button>
+</div>
+<div *ngIf="!mode then numbers;else classTemplate">
+
+</div>
```
Comment (DmMov) [inline src/app/components/main-schedule/main-schedule.component.html]: 5

**Review by Redingston — APPROVED** [review]

**Review by vunt1k — APPROVED** [review]

File: src/app/components/main-schedule/main-schedule.component.scss [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  text-align: center;
+  align-self: center;
+  }
+
+  .selected {
+    width: 4rem;
+    color: $accent;
+    align-self: center;
+    margin: 1rem auto;
+
```
Comment (bodkia22) [inline src/app/components/main-schedule/main-schedule.component.scss:36]: What it is ?

**Review by bodkia22 — APPROVED** [review]

**Pull request merged**